docs/developer/extending/apps/subscription-webhook-payloads.mdx

```diff
@@ -72,28 +72,67 @@ mutation {
 }
 ```
 
+Example **asynchronous** webhook payload:
 
-Payload generated from subscription will have all the fields that you asked for, plus some meta fields.
+```json
+{
+  "product": {
+    "id": "UHJvZHVjdDo3Ng==",
+    "name": "Coconut Juice"
+  }
+}
+```
 
-Example payload:
+Example **synchronous** webhook create mutation for subscription payload:
+
+```graphql
+mutation {
+  webhookCreate(
+    input: {
+      app: "<app_id>"
+      name: "test_payment_capture_subscription"
+      targetUrl: "<webhook_url>"
+      syncEvents: [PAYMENT_CAPTURE]
+      query: "subscription{event{...on PaymentCaptureEvent{payment{id created}}}}"
+    }
+  ) {
+    errors {
+      field
+      message
+    }
+    webhook {
+      id
+    }
+  }
+}
+```
+
+Example **synchronous** webhook payload:
 
 ```json
 {
-    "product": {
-      "id": "UHJvZHVjdDo3Ng==",
-      "name": "Coconut Juice"
-    },
-    "meta": {
-      "issuedAt": "2022-03-23T14:37:27.084275+00:00",
-      "version": "dev",
-      "issuingPrincipal": {
-        "id": "<id>",
-        "type": "user"
-      }
-    }
+  "payment": {
+    "id": "UGF5bWVudDoyNDE=",
+    "created": "2022-08-03T08:52:04.202449+00:00"
   }
+}
 ```
 
+## Synchronous webhooks payload
+
+Defining synchronous webhooks payloads using subscriptions is also supported. Each event subscription type has access to its specific objects fields.
+Keep in mind that some fields that are using synchronous event to be resolved are not allowed to be called in synchronous webhook payload subscriptions.
```

Comment:
```suggestion
Keep in mind that some fields that are using the synchronous event to be resolved are not allowed to be called in synchronous webhook payload subscriptions.
```

Reply:
Also, shouldn't this sentence be in the warning block?

Reply:
```suggestion
:::caution
Keep in mind that some fields that are using the synchronous event to be resolved are not allowed to be called in synchronous webhook payload subscriptions.
:::
```

Reply:
true